components/eventing-controller/README.md

```diff
@@ -70,6 +70,7 @@ This section explains how to use the Eventing Controller. It expects the followi
 |  `JS_STREAM_RETENTION_POLICY`     | The policy to delete events from the stream: `limits` or `interest`. See https://docs.nats.io/using-nats/developer/develop_jetstream/model_deep_dive#stream-limits-retention-and-policy. |
 |  `JS_STREAM_MAX_MSGS`             | The maximum number of messages in the stream. Used only when storage policy is set to `limits`. |
 |  `JS_STREAM_MAX_BYTES`            | The maximum size of the stream in bytes. Used only when storage policy is set to `limits`.     |
+|  `JS_STREAM_SUBJECT_PREFIX`       | The prefix for the JetStream stream subjects filter. Used only when `EVENT_TYPE_PREFIX` is empty.     |
```

Comment:
```suggestion
|  `JS_STREAM_SUBJECT_PREFIX`       | The prefix for the JetStream subjects filter. Used only when `EVENT_TYPE_PREFIX` is empty.     |
```
Do we need to repeat the word `stream` since it's already included in `JetStream`?